ppa/common/fixtures/sites.json

```diff
@@ -0,0 +1 @@
+[{"model": "sites.site", "pk": 1, "fields": {"domain": "localhost:8000", "name": "Princeton Prosody Archive"}}]
```

Comment:
I suggest naming the fixture more specifically so it's more obvious what this is used for - test_site? ci_test_site?